_data/projects/strongbox.yml

```diff
@@ -0,0 +1,18 @@
+name: Strongbox
+desc: An alternative Java-based artifact repository manager.
+site: https://github.com/strongbox/strongbox/
+tags:
+- java
+- springframework
+- orientdb
+- restassured
+- jetty
+- junit
+- maven
+- nuget
+- artifact repository manager
+
+upforgrabs:
+  name: good first issue
+  link: https://github.com/strongbox/strongbox/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20label%3A%22good%20first%20issue%22%20label%3A%22help%20wanted%22%20
```

Comment:
This could be simplified to `https://github.com/strongbox/strongbox/labels/good%20first%20issue` unless you want to cover multiple labels

Reply:
Hi,

I agree, but I was thinking of including both labels in the query. The "help wanted" label doesn't necessarily mean that the issue is a "good first issue". Sometimes people are up for more challenging tasks.

I could change it, if you think it will be better, though!

Reply:
@carlspring All the good first issues are already help wanted issues.
I advice https://github.com/strongbox/strongbox/labels/good%20first%20issue  or https://github.com/strongbox/strongbox/labels/help%20wanted

Reply:
Okay, as per your request, @shiftkey and @ritwik12, I have fixed this.

Could you kindly accept this pull? Many thanks in advance! :)

Reply:
Yes sure

Reply:
Thanks! :)